apps/schematic/api/schematic_api/test/test_synapse_endpoints.py

```diff
@@ -38,6 +38,29 @@
 }
 
 
+@pytest.mark.synapse
+@pytest.mark.secrets
+class TestGenerateGoogleSheetManifests(BaseTestCase):
+    """Tests google sheet manifest endpoint"""
+
+    def test_success(self) -> None:
+        """Test for successful result"""
+        url = (
+            f"/api/v1/generateGoogleSheetManifests?schemaUrl={TEST_SCHEMA_URL}"
```

Comment:
I don't think here the API is interacting with synapse because dataset_id is not used as a parameter